libs/ledger-live-common/src/families/cosmos/js-buildTransaction.ts

```diff
@@ -21,17 +21,13 @@ import { PubKey } from "@keplr-wallet/proto-types/cosmos/crypto/secp256k1/keys";
 import { AuthInfo, Fee } from "@keplr-wallet/proto-types/cosmos/tx/v1beta1/tx";
 import { TxBody } from "cosmjs-types/cosmos/tx/v1beta1/tx";
 import Long from "long";
-import { Coin } from "@keplr-wallet/proto-types/cosmos/base/v1beta1/coin";
-import BigNumber from "bignumber.js";
-import { EncodeObject, GeneratedType, makeAuthInfoBytes, Registry } from "@cosmjs/proto-signing";
-import { CosmosAPI } from "./api/Cosmos";
 
 type ProtoMsg = {
   typeUrl: string;
   value: Uint8Array;
 };
 
-export const buildTransaction = async (
+export const txToMessages = async (
```

Comment:
TBC, but I think this may not need to be `async`

Reply:
Done